lite/examples/image_segmentation/android/app/build.gradle

```diff
@@ -4,7 +4,6 @@ apply plugin: 'kotlin-android-extensions'
 
 android {
     compileSdkVersion 29
-    buildToolsVersion "29.0.0"
```

Comment:
Why do you remove this?

Reply:
Because of this: https://developer.android.com/studio/releases/gradle-plugin#3-1-0